examples/networking/router_route/base-compute-port.tf

```diff
@@ -0,0 +1,13 @@
+resource "vkcs_networking_port" "port-for-compute-tf-example" {
```

Comment:
we do not need this

Reply:
but we need this later for port examples

Reply:
removed